### PR TITLE
Dynamic sidebar

### PR DESCRIPTION
**Pull request by hadim**
I can get pt-content to slide when user put mouse hover sidebar but I can't get pt-page-header to follow the move (I think it comes from `position: fixed`).

Any help would be appreciated !

### CODE REVIEW

**Comment by hadim** [comment]
I fixed the issue. Good to merge for me.

File: src/coffee/settings/settings.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <section flex flex-order="2" layout="column" id="middle-col">
     <!-- Settings Title Container -->
-    <pt-page-header layout="row" go-back="settings.goBack()"></pt-page-header>
+    <pt-page-header layout="row" title="Settings" go-back="settings.goBack()"></pt-page-header>
```
Comment (hadim) [inline src/coffee/settings/settings.html:3]: Title is not displayed. Any idea why ?

**Comment by hadim** [comment]
NOT READY TO MERGE

File: src/coffee/settings/settings.html (hunk 1 above)
Comment (BoLaMN) [inline src/coffee/settings/settings.html:3]: "'Settings'" maybe

Comment (hadim) [inline src/coffee/settings/settings.html:3]: Great catch !

**Pull request merged**